### ISSUE TITLE
Framed Wires and Draw Plates uncraftable

### ISSUE THREAD

**Issue by katzenpapst** [MrTJP/ProjectRed#770]
Just noticed that I can't craft Framed Wires, both normal and insulated. Tested it with the draw plate, it doesn't work either. Both show up as recipes in NEI, though. I suspect something is broken regarding Forge Multipart. To exclude mod interations, I tested with a mini setup:

```
1.7.10/CodeChickenLib-1.7.10-1.1.3.127-universal.jar
1.7.10/ForgeMultipart-1.7.10-1.1.2.331-universal.jar
1.7.10/MrTJPCore-1.7.10-1.0.5.12-universal.jar
CodeChickenCore-1.7.10-1.0.4.35-universal.jar
neiaddons-mc1710-1.12.3.11.jar
NotEnoughItems-1.7.10-1.0.4.95-universal.jar
ProjectRed-1.7.10-4.5.16.77-Base.jar
ProjectRed-1.7.10-4.5.16.77-Compat.jar
ProjectRed-1.7.10-4.5.16.77-Integration.jar
ProjectRed-1.7.10-4.5.16.77-Lighting.jar
ProjectRed-1.7.10-4.5.16.77-Mechanical_beta.jar
ProjectRed-1.7.10-4.5.16.77-World.jar
```

**Comment by zISKo666** [comment]
I have the exact same problem since I upgraded to a higher version than ForgeMultipart-1.7.10-1.1.1.326-universal.jar

**Comment by Keridos** [comment]
Yes having the same issue here.

**Comment by Andreiovitch** [comment]
I have the same issue with ProjectRed 4.5.16 build #77, FMP 1.1.2.331, Forge .1291

**Comment by juniorjunior** [comment]
Until this is addressed I wrote a MineTweaker script to fix this: http://pastebin.com/hPyGPZLH

**Comment by MrTJP** [comment]
Fixed via 9f57d7f0d8c5cb60e5bf5a11db00301dbd28afe4
